lnbits/wallets/cliche.py

```diff
@@ -157,19 +184,30 @@ async def get_payment_status(self, checking_id: str) -> PaymentStatus:
             payment.get("preimage"),
         )
 
+    async def on_message(self, message: str):
+        try:
+            data = json.loads(message)
+            msg_id = data.get("id")
+            if msg_id is not None and msg_id in self.futures:
+                future = self.futures.pop(msg_id)
+                future.set_result(data)
+            elif data.get("result", {}).get("status"):
+                yield data["result"]["payment_hash"]
```

Comment:
isn't it possible to fall through here without anything happening in if, elif?

Reply:
as intended i guess! if it's not a paid invoice message, or a response from a specific command sent to cliche, we don't care for it!